api/editor/api-editor/src/main/java/ca/bc/gov/tno/areas/reports/controllers/CBRAController.java

```diff
@@ -29,7 +29,7 @@
 /**
  * ActionController class, provides endpoints for actions.
  */
-@RolesAllowed({ "administrator", "editor" })
+@RolesAllowed({ "administrator" })
```

Comment:
Only admins can run the report.  I'd like to figure out how to only apply these rules to endpoints not the whole controller.